src/components/App.js

```diff
@@ -77,6 +86,10 @@ export default function App() {
     getRecipes()
   }, [url])
 
+  useEffect(() => {
```

Comment:
You might want to consider moving this from here to a useLocalStorage.js custom hook